log/log.go

```diff
@@ -0,0 +1,42 @@
+package log
```

Comment:
Similar to init.go https://sourcegraph.com/github.com/sourcegraph/sourcegraph/-/blob/lib/log/init.go. However, I have moved logic to panic to the `internal/logger`. i.e.  Get() chooses to allow `safeGet` based on `DevMode`, but otherwise let internal/logger decide if/when to panic and how to handle `safeGet`.